### ISSUE TITLE
Unable to enable/disable CPU and memory HotPlug with govc

### ISSUE THREAD

**Issue by angystardust** [vmware/govmomi#1271]
I will be very useful to enable CPU and Memory HotPlug with govc
I only find a way to check if it's enabled or not using this command:
```
govc object.collect -s "/PRD/vm/myvm01" config.memoryHotAddEnabled
govc object.collect -s "/PROD/vm/myvm01" config.cpuHotAddEnabled
```

**Comment by angystardust** [comment]
Ok, i've found a way to do that by setting 2 extra variables:
`govc vm.change -vm myvm01 -e vcpu.hotadd=true -e mem.hotadd=true`
So it's just a matter of documenting how to do this.
@dougm if you agree, i can submit a patch to the USAGE.md file

**Comment by dougm** [comment]
Fixed by #1272
